src/drivers/imx/ipc.c

```diff
@@ -29,36 +29,6 @@ struct ipc_data {
 	struct ipc_data_host_buffer dh_buffer;
 };
 
```

Comment:
Should this be done in same commit as patch that removes the "cb" callback. I think bisect won't work otherwise.

Typo in commit: "Clean up IPC drivers by removing..."

Reply:
It's different callback.

Reply:
Ack @tlauda